content/docs/ops/configuration-management.md

```diff
@@ -47,8 +47,6 @@ is encouraged and qualifies as a review. Review should include assessment of arc
 
 ![Pipeline Example](/img/pipeline-example.png)
 
-A more detailed example of this process can be seen in [Updating Cloud Foundry]({{< relref "updating-cf.md" >}}).
```

Comment:
@brittag Do we need to worry about any :compliance: issues when dropping these old, out of date docs since they are mentioned in our config management plan?